packages/fast-layouts-react/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -0,0 +1,311 @@
+import * as React from "react";
+import { get } from "lodash-es";
+import manageJss, { ComponentStyles, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
+import Foundation, { IFoundationProps } from "../foundation";
+import { HorizontalOverflowProps } from "./horizontal-overflow.props";
+import { getItems, scrollLeft } from "./horizontal-overflow.utilities";
```

Comment:
Based on earlier conversations this seem a bit more like a component than a layout - thoughts?

Reply:
@chrisdholt do you want to weigh in here? The idea is also still nebulous to me. Once we determine criteria we should file an issue to document the differences so we can offer more clarity in future. If we determine this is more of a component library item I can move it to base, I think it would make more sense.

Reply:
@nicholasrice can you expand on that? I feel like this really just serves to facilitate a specific layout for UI elements.

Reply:
I'm open to having this live as a component in base. It would provide more control and open up the design system values.